ansible/library/dual_tor_facts.py

```diff
@@ -119,7 +120,9 @@ def main():
             hostvars=dict(required=True, default=None, type='dict'),
             vm_config=dict(required=True, default=None, type='dict'),
             port_alias=dict(required=True, default=None, type='list'),
-            vlan_intfs=dict(required=True, default=None, type='list')
+            vlan_intfs=dict(required=True, default=None, type='list'),
+            vlan_config=dict(required=False, default=None, type='str'),
+            working_dir=dict(required=False, default='.', type='str'),
```

Comment:
what is the use of working_dir ?

Reply:
This is typically run in the ansible directory. Now I need to run it in tests directory. The input working_dir is for the script to adjust paths accordingly.